docs/en/seatunnel-engine/deployment.md

```diff
@@ -235,9 +235,12 @@ hazelcast-client:
 
 ## 7. Start SeaTunnel Engine Server Node
 
+Can be started by a daemon with `-d`.
+
 ```shell
 mkdir -p $SEATUNNEL_HOME/logs
-nohup bin/seatunnel-cluster.sh 2>&1 &
+touch $SEATUNNEL_HOME/logs/seatunnel-server.out
```

Comment:
Why not put it in your shell command, to reduce operation which user need do.

Reply:
Well, I think so